pyseir/ensembles/ensemble_runner.py

```diff
@@ -0,0 +1,289 @@
+import datetime
+import logging
+import os
+import numpy as np
+import json
+import copy
+from collections import defaultdict
+from functools import partial
+from multiprocessing.pool import Pool
+from pyseir.models.seir_model import SEIRModel
+from pyseir.parameters.parameter_ensemble_generator import ParameterEnsembleGenerator
+from pyseir.models.suppression_policies import generate_empirical_distancing_policy
+from pyseir import OUTPUT_DIR
+from pyseir import load_data
+from pyseir.inference import fit_results
+from pyseir.reports.county_report import CountyReport
+
+
+compartment_to_capacity_attr_map = {
+    'HGen': 'beds_general',
+    'HICU': 'beds_ICU',
+    'HVent': 'ventilators'
+}
+
+
+class EnsembleRunner:
```

Comment:
This is super small, but I'd recommend inheriting from `object`.  the [google style guide](https://google.github.io/styleguide/pyguide.html#39-classes) give a good explanation - basically you get some better build-in behaviors

Reply:
This is python 2. In python 3 that is implicit so is no longer recommended.